Userland/DevTools/HackStudio/HackStudioWidget.cpp

```diff
@@ -294,7 +294,6 @@ bool HackStudioWidget::open_file(const String& full_filename)
     current_editor().set_focus(true);
 
     current_editor().on_cursor_change = [this] { update_statusbar(); };
-    current_editor().on_change = [this] { update_gml_preview(); };
```

Comment:
IMO updating the GML preview on every change is a feature we want to have.

How about changing the API of the `EditorWrapper` and allow registering a callback function
that would get called on every editor change?
Then `EditorWrapper` could still internally take care of marking the document as dirty, and `update_gml_preview()` can get called regardless.